modules/http-helper/http_helper_test.go

```diff
@@ -165,6 +170,8 @@ func retryHandler(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusServiceUnavailable)
```

Comment:
Can you read the body here as well, so that we read from the body multiple times? I believe the HTTP functions read it internally as well, but this at least makes it explicit.

Reply:
Added the readAll here :+1: